### ISSUE TITLE
Implement Firebase Firestore automatic flyer upload system

### ISSUE THREAD

**Issue by bsalgado98** [StevensSEC/sec.club#32]
https://stackoverflow.com/questions/32702431/display-images-fetched-from-s3

**Comment by dyc3** [comment]
One Firebase's services might have a more elegant solution.

**Comment by bsalgado98** [comment]
Yeah you’re right! I almost forgot about Firestore. If we’re already doing a fire base implementation for hosting, it makes more sense to just use Firebase tools for this job. Good catch 👌🏼

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by carpenterd777** [comment]
Alright, I attempted trying to implement this feature and after investigation I have determined that this is outside the scope of my knowledge for me to implement. I set up a database on Firebase (which redirected me to GCP) if anyone else wants to give it a shot. 

These links might be helpful:
[Get started](https://firebase.google.com/docs/firestore/quickstart)
[Get data](https://firebase.google.com/docs/firestore/query-data/get-data)

**Comment by dyc3** [comment]
I'd like to avoid a database if possible. I don't think we need it. We should just be able to list the flyers from the storage bucket and grab them from there.

**Comment by dyc3** [comment]
The real question is whether or not this is actually feasible or actually needed.

1. Do we want metadata (title, date, etc.) about the event available?
   a. If yes, do we enforce a file naming convention so we can extract that metadata as needed? Do we invent some custom format to store the metadata and the picture?
2. Why not treat events like we treat articles? The obvious answer is so new flyers can be uploaded without editing code, but do we really need that to be the case? We have automated deployment, and we could streamline adding new flyers like we did with articles so it's easy. I think it's reasonable to assume future eboards will be able to code.
